sysbuild/Kconfig

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+comment "Sysbuild image configuration"
+
+osource "$(BOARD_DIR)/Kconfig.sysbuild"
+
+config WARN_EXPERIMENTAL
+	bool
+	prompt "Warn on experimental usage"
+	help
+	  Print a warning when the Kconfig tree is parsed if any experimental
+	  features are enabled.
+
+config SUPPORT_BOOTLOADER
+	bool
+	default y
+
+config SUPPORT_BOOTLOADER_MCUBOOT_ZEPHYR
+	bool
+	default y
+
+choice BOOTLOADER
+	prompt "Bootloader support"
+	default BOOTLOADER_NONE
+	depends on SUPPORT_BOOTLOADER
+
+config BOOTLOADER_NONE
+	bool "None"
+	help
+	  Do not Include a bootloader in the build
+
+config BOOTLOADER_MCUBOOT
+	bool "MCUboot"
+	depends on SUPPORT_BOOTLOADER_MCUBOOT_ZEPHYR
```

Comment:
I'm aware the primary use case of sysbuild is to support flashing a bootloader and application in one step, but why include MCUBoot specific KConfig values in the primary sysbuild KConfig? Shouldn't these be moved to a MCUBoot specific Kconfig file?

Reply:
Good point.

Will move this to dedicated `Kconfig.bootloader` file\ to have a common location for bootloader directly known and supported across various platforms for Zephyr. This will make the Kconfig somewhat generic.

As background info, it was originally placed here because it corresponds to the location in the toplevel Kconfig.zephyr file:
https://github.com/zephyrproject-rtos/zephyr/blob/d74cb2a3df2376dd65936d30cf4dc56092044761/Kconfig.zephyr#L633-L634

but let's avoid the cluttering mistakes of the past.
(Maybe we should even do some cleanup to `Kconfig.zephyr` although in a different PR)